### PR TITLE
[azure-devops-exporter] Allow to build area path from project name only

### LINKED ISSUE #3076: The 'System.AreaPath' error during exporting test cases to Azure DevOps when the Area name is inherited from the Project name
**Expected behavior**
The test cases are exported to Azure DevOps work items

**Artifacts**
[export_azure.txt](https://github.com/vividus-framework/vividus/files/9372360/export_azure.txt)

**Execution environment (please complete the following information):**
 - Vividus version: 0.4.11
 - Exporter version: 0.4.11

**Additional context**
#project ID or project name (Project Settings)
azure-devops-exporter.project=TestName
#the area of the product which a test case is associated with
azure-devops-exporter.area=TestName

### PR DESCRIPTION
**Pull request by uarlouski**
Fixes #3076

### CODE REVIEW

**Review by ikalinin1 — APPROVED** [review]

File: docs/modules/integrations/pages/azure-devops-exporter.adoc [hunk 1]
```diff
@@ -38,10 +38,12 @@ Please make sure that the `bdd.configuration.formats` property includes JSON val
...
 |Project ID or project name.
 
-|`*azure-devops-exporter.area*`
+|`azure-devops-exporter.area`
 |
 |string
-|The area of the product which a test case is associated with.
+a|The area of the product which a test case is associated with.
+
+If in AzureDevOps the Area name is inherited from the Project name this property should be empty.
```
Comment (valfirst) [inline docs/modules/integrations/pages/azure-devops-exporter.adoc:46]:
```suggestion
If the Area name is inherited from the Project name in AzureDevOps this property should be empty.
```

Comment (valfirst) [inline docs/modules/integrations/pages/azure-devops-exporter.adoc:46]: How can users identify it's inherited?

File: vividus-to-azure-devops-exporter/src/main/java/org/vividus/azure/devops/facade/AzureDevOpsFacade.java [hunk 2]
```diff
@@ -281,8 +281,12 @@ private AddOperation createDescriptionOperation(String value)
     private List<AddOperation> createCommonOperations(String testTitle)
     {
         List<AddOperation> operations = new ArrayList<>();
-        operations.add(new AddOperation("/fields/System.AreaPath", format("%s\\%s", options.getProject(),
-                options.getArea())));
+        String areaPath = options.getProject();
+        if (options.getArea() != null)
```
Comment (valfirst) [inline vividus-to-azure-devops-exporter/src/main/java/org/vividus/azure/devops/facade/AzureDevOpsFacade.java:285]:
```java
private Optional<String> area;
```
?

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus-to-azure-devops-exporter/src/main/java/org/vividus/azure/devops/facade/AzureDevOpsFacade.java (hunk 2 above)
Comment (uarlouski) [reply vividus-to-azure-devops-exporter/src/main/java/org/vividus/azure/devops/facade/AzureDevOpsFacade.java:285]: Spring won't set optional empty into this

**Review by valfirst — APPROVED** [review]

**Pull request merged**